roles/kafka_controller/tasks/kraft_tasks.yml

```
@@ -0,0 +1,15 @@
+---
+- name: get uuid
+  shell: "{{ binary_base_path }}/bin/kafka-storage random-uuid"
+  environment:
+    KAFKA_OPTS: "-Xlog:all=error -XX:+IgnoreUnrecognizedVMOptions"
+  register: uuid_key
+  run_once: true
+
+- name: get CLUSTER_ID
+  set_fact:
+    clusterid: "{{ uuid_key.stdout }}"
+
+- name: format meta directory
+  shell: "{{ binary_base_path }}/bin/kafka-storage  format -t {{ clusterid }} -c {{ kafka_controller.config_file }}  --ignore-formatted"
+  register: format_meta
```

```suggestion
- name: format meta directory
  shell: "{{ binary_base_path }}/bin/kafka-storage  format -t {{ clusterid }} -c {{ kafka_controller.config_file }}  --ignore-formatted"
  register: format_meta
  with_fact:
    clusterid: "{{ uuid_key.stdout }}"
```

can you change the task names to use capital letters